lib/prometheus.sh

```diff
@@ -21,11 +21,11 @@
 declare -r PROMETHEUS_OPERATOR_VERSION=${PROMETHEUS_OPERATOR_VERSION:-v0.11.0}
 declare -r PROMETHEUS_REPLICAS=${PROMETHEUS_REPLICAS:-1}
 
-# constants
-
 # constants
 declare -r KUBE_PROM_DIR="$PROJECT_ROOT/tmp/kube-prometheus"
 declare -r MONITORING_NS="monitoring"
+declare DASHBOARD_DIR="$KUBE_PROM_DIR/grafana-dashboards"
+declare KEPLER_EXPORTER_GRAFANA_DASHBOARD_JSON=$( curl -fsSL https://raw.githubusercontent.com/sustainable-computing-io/kepler/main/grafana-dashboards/Kepler-Exporter.json | sed '1 ! s/^/         /' )
```

Comment:
I opted for curling the Kepler-Exporter dashboard from the kepler repo directly rather than using a local file as in theory you could clone this repo and run the scripts without cloning the kepler repo.

If we think that a local path is more suitable I can add some checks to see if there's a local file, if not then curl the repo file...